core/src/main/java/io/grpc/internal/AbstractManagedChannelImplBuilder.java

```diff
@@ -378,7 +378,6 @@ public ManagedChannel build() {
       // First interceptor runs last (see ClientInterceptors.intercept()), so that no
       // other interceptor can override the tracer factory we set in CallOptions.
       effectiveInterceptors.add(0, censusStats.getClientInterceptor());
-      //      }
```

Comment:
This will be a style error:
[ant:checkstyle] [ERROR] /usr/local/google/home/songya/Google/grpc-java-cloned/grpc-java/core/src/main/java/io/grpc/internal/AbstractManagedChannelImplBuilder.java:381: Comment has incorrect indentation level 6, expected is 31, indentation should be the same level as line 380. [CommentsIndentation]

Not sure how to fix it so I just removed this line.

Reply:
That's weird.  I wonder why it says the comment should be indented 31 levels.  I can try to add that code back next, so I think this is okay to merge.